lang/en/tool_mergeusers.php

```diff
@@ -186,8 +186,17 @@
     data from new user (by default). This also means that data from old user is 
     deleted to keep the consistence. Otherwise, if you uncheck this option, 
     data from old user will be kept.';
-
 $string['starttime'] = 'Started merging at {$a}';
 $string['finishtime'] = 'Finished merging at {$a}';
 $string['timetaken'] = 'Merge took {$a} seconds';
 $string['privacy:metadata'] = 'The Merge User Accounts plugin does not store any personal data.';
+$string['get_queue_merging_requests'] = 'Get current queue of user profiles merging requests and launch adhoc task for each single request';
+$string['cannotfindusertoremove'] = 'Cannot find the user to remove. Change the input parameters.';
```

Comment:
these strings should be updated according to the parameters, if you like my proposal.

Reply:
done